### PR TITLE
Eks distro base

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

**Review by bnrjee — APPROVED** [review]

File: release/prow-release.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ fi
 BASE_DIRECTORY=$(git rev-parse --show-toplevel)
 cd ${BASE_DIRECTORY}
 AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
-export BASE_REPO=${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com
+export BASE_REPO=316434458148.dkr.ecr.us-west-2.amazonaws.com/eks-distro/base
```
Comment (bnrjee) [inline release/prow-release.sh:29]: `export BASE_REPO=316434458148.dkr.ecr.us-west-2.amazonaws.com`

Comment (bnrjee) [reply release/prow-release.sh:29]: Not sure, if we want to keep the string, "eks-distro/base"?

Comment (TerryHowe) [reply release/prow-release.sh]: done

**Review by bnrjee — APPROVED** [review]

**Pull request merged**